ext/standard/var_unserializer.re

```diff
@@ -23,7 +23,6 @@
 
 /* {{{ reference-handling for unserializer: var_* */
 #define VAR_ENTRIES_MAX 1018     /* 1024 - offsetof(php_unserialize_data, entries) / sizeof(void*) */
-#define VAR_DTOR_ENTRIES_MAX 255 /* 256 - offsetof(var_dtor_entries, data) / sizeof(zval) */
```

Comment:
Might we want to change the structure definition to use correct constant instead?

Reply:
Might be better. Done.